addons/mail/models/mail_message.py

```diff
@@ -102,6 +102,8 @@ def default_get(self, fields):
         'mail.notification', 'mail_message_id', 'Notifications',
         auto_join=True, copy=False, depends=['notified_partner_ids'])
     # user interface
+    mentioned_partner_ids = fields.Many2many(
+        'res.partner', 'mail_message_res_partner_mentioned_rel', string='Mentioned Partners')
```

Comment:
Don't we want to simply rely on existing `partner_ids` for now?

Mentions and recipients are completely mixed up currently anyway, it doesn't make sense to create a new field if they are not completely split everywhere else too.

Reply:
As we are struggling to remove fields on mail.message, let us avoid adding an unnecessary one indeed :)